terraso_backend/apps/graphql/schema/visualization_config.py

```diff
@@ -144,6 +145,18 @@ def get_queryset(cls, queryset, info):
     def resolve_mapbox_tileset_id(self, info):
         if self.mapbox_tileset_id is None:
             return None
+
+        shared_resources = self.data_entry.shared_resources.all()
+        users = []
+        for shared_resource in shared_resources:
+            members = (
+                shared_resource.target.membership_list.members.all()
+            )  # TODO Get only approved memberships
+            ids = [member.id for member in members]
+            users = set(users + ids)
+        for user_id in users:
+            notify_user(user_id, f"Updated: {self.mapbox_tileset_id}")
```

Comment:
question: should the message be structured data?

suggestion: indicate that a _tileset_ was updated. we may have other kinds of updates in the future.